x-pack/plugins/reporting/index.d.ts

```diff
@@ -13,6 +13,8 @@ import {
   NotificationsStart,
 } from '../../../src/core/public';
 
+export { ToastsSetup, HttpSetup, ApplicationStart, IUiSettingsClient } from 'src/core/public';
```

Comment:
Let's try to keep this file super lean, and have server things in `reporting/server/types.d.ts` and UI stuff in `reporting/public/types.d.ts`. But I don't think we need this export here anyway, because our UI libs are in NP and they can import from `src/core/public`